bugsnag-android-core/src/test/java/com/bugsnag/android/BreadcrumbSerializationTest.kt

```diff
@@ -17,7 +17,7 @@ internal class BreadcrumbSerializationTest {
             val timestamp = Date(0)
             return generateSerializationTestCases(
                 "breadcrumb",
-                Breadcrumb("hello world", BreadcrumbType.MANUAL, mutableMapOf(), timestamp, NoopLogger),
+                Breadcrumb("helloworld", BreadcrumbType.MANUAL, mutableMapOf(), timestamp, NoopLogger),
```

Comment:
I'm not sure why this has changed?

Reply:
The JSON comparison as it currently stands on this branch removes *all* whitespace before comparing.
The issue has been fixed on `next` (https://github.com/bugsnag/bugsnag-android/pull/1512) and this code will change back before we merge the integration branch down.

But to keep the rebase / PRs simple I opted to simply change the same model to match the data in the JSON fixtures. Once the integration branch is ready to be merged into next - these tests will start failing and need to be fixed "back" again.